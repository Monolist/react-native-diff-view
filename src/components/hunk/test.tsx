import React from 'react';
import renderer from 'react-test-renderer';
import CodeCell from './code_cell';

describe('components/hunk', () => {
  describe('CodeCell', () => {
    it('renders correctly', () => {
      expect(renderer.create(<CodeCell text="A" />).toJSON()).toMatchSnapshot();
    });
  });
});
