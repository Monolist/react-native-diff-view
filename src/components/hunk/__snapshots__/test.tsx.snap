// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/hunk CodeCell renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "paddingHorizontal": 6,
      "paddingVertical": 3,
    }
  }
>
  <Text
    numberOfLines={1}
    style={
      Object {
        "fontFamily": "Courier New",
        "fontSize": 12,
      }
    }
  >
    A
  </Text>
</View>
`;
